### ISSUE TITLE
Some assets hasn't been created in the publish folder

### ISSUE THREAD

**Issue by titomncl** [titomncl/Odin#69]
I don't have any clue of what the problem is, you'll need a lot of investigation then.

**Comment by titomncl** [comment]
```
2022-05-09 09:35:28 -- [WARNING] -- X:/VSPA/OUT/SEQ/S170/S170P120/FX: Folder exists
2022-05-09 09:35:28 -- [WARNING] -- X:/VSPA/OUT/SEQ/S170/S170P120/FX/BANNER_01: Folder exists
```

**Comment by titomncl** [comment]
This issue has been resolved with the pull request #71.
